object/user_api_token.go

```diff
@@ -164,3 +167,39 @@ func MakeUserApiToken(user *User) UserApiToken {
 		ApiToken: user.AccessKey + user.AccessSecret,
 	}
 }
+
+func GetUserTokens(user *User) ([]User, error) {
+	var tokens []User
+
+	tagPattern := fmt.Sprintf("<access-token><access-token-user-id:%s>", user.Id)
+
+	err := ormer.Engine.Where("tag LIKE ?", tagPattern).Find(&tokens)
+	if err != nil {
+		return nil, err
+	}
+
+	return tokens, nil
+}
+
+func UpdateTokens(tokens []User, columns []string) error {
+	session := ormer.Engine.NewSession()
```

Comment:
we have Repo with TransactionManager manager, check opportunity to reuse.

Reply:
done